libraries/ms-helper/rules.mk

```diff
@@ -9,7 +9,7 @@
 $(T)_DEPS := ms-common
 
 ifeq (x86,$(PLATFORM))
-$(T)_EXCLUDE_TESTS := mcp2515
```

Comment:
Now that x86 has an adc implementation, you shouldn't need this?

Reply:
Avery was hitting a lot of issues getting travis to run the test successfully, so I made the call to just skip it on x86. I think it's alright, hopefully doesn't cause issues down the line.